app/views/admin/users/index.html.haml

```diff
@@ -10,7 +10,7 @@
     %table.table.table-striped.table-bordered.table-hover.datatable
       %thead
         %th ID
-        %th State
+        %th Confirmation State
```

Comment:
I would suggest to keep it that way. The Bootstrap-Switch-Checkbox would simply toggle between 'Yes' and 'No'. Changing a state of user to 'Yes' or 'No' doesn't convey much.